spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +75,21 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	public void setArguments(Map<Integer, Map<String, String>> arguments) {
+		this.arguments = arguments;
+	}
+
+	public Map<Integer, Map<String, String>> getArguments() {
+		return arguments;
+	}
+
+	public List<ModuleLaunchRequest> asModuleLaunchRequests() {
```

Comment:
I still feel like this logic should be moved out, probably into the ModuleLauncher.